docs/getting_started.md

```diff
@@ -185,12 +179,70 @@ ssh-keygen -t ed25519 -f /mnt/secrets/ssh/id_ed25519 -N ""
 
 Either way, make sure to configure the private SSH key path in the SSH settings on the frontend. In this case, the path should be set to `/mnt/secrets/ssh/id_ed25519`.
 
-## Deployment
+## Addons
+
+### SSV-node
+The [ssv.network](https://ssv.network/overview/) is a fully decentralized, open-source, and trustless DVT Network that provides a reusable infrastructure solution for decentralizing Ethereum validators.
+
+1. Generate a key pair
+
+    ```shell
+    nix run .#init-ssv -- <hostname>
+    ```
+
+2. Transfer the private key securely to the target machine to the path you have configured at the frontend
+
+3. Register an SSV operator
+
+    You can register an operator on the Web UI. For more information, see [these instructions](https://ssv-network.gitbook.io/guides/operator/registering-an-operator) by ssv.network.
```

Comment:
"on the web ui", which one does this mean, ours or theirs? probably good to mention explicit options for both